src/components/OrderDetailWidget/OrderDetailWidget.tsx

```diff
@@ -143,25 +146,26 @@ const OrderDetailWidget: FC<OrderDetailWidgetProps> = ({ order }) => {
         baseTokenInfo={signerToken}
         maxAmount={null}
         side={userIsMakerOfSwap ? "sell" : "buy"}
+        tradeNotAllowed={walletChainIdIsDifferentThanOrderChainId}
         quoteAmount={senderAmount || "0.00"}
         quoteTokenInfo={senderToken}
         onBaseAmountChange={() => {}}
         onChangeTokenClick={() => {}}
         onMaxButtonClick={() => {}}
       />
-      {!tokenInfoLoading && (
-        <StyledInfoButtons
-          isMakerOfSwap={userIsMakerOfSwap}
-          isIntendedRecipient={userIsIntendedRecipient}
-          isExpired={orderIsExpired}
-          isNotConnected={!active}
-          token1={signerToken?.symbol}
-          token2={senderToken?.symbol}
-          rate={tokenExchangeRate}
-          onFeeButtonClick={toggleShowFeeInfo}
-          onCopyButtonClick={handleCopyButtonClick}
-        />
-      )}
+      <StyledInfoButtons
+        isExpired={orderIsExpired}
+        isDifferentChainId={walletChainIdIsDifferentThanOrderChainId}
+        isIntendedRecipient={userIsIntendedRecipient}
+        isMakerOfSwap={userIsMakerOfSwap}
+        isNotConnected={!active}
+        orderChainId={orderChainId}
+        token1={signerToken?.symbol}
+        token2={senderToken?.symbol}
+        rate={tokenExchangeRate}
+        onFeeButtonClick={toggleShowFeeInfo}
+        onCopyButtonClick={handleCopyButtonClick}
+      />
       <ActionButtons
         isExpired={orderIsExpired}
         isMakerOfSwap={userIsMakerOfSwap}
```

Comment:
```suggestion
        isMakerOfSwap={userIsMakerOfSwap}
        isDifferentChainId={walletChainIdIsDifferentThanOrderChainId}
```

then this isDifferentChainId can appended to buttonDisabled with an OR operator